### ISSUE TITLE
Show ligatures in examples

### ISSUE THREAD

**Issue by mtelesha** [chrissimpkins/codeface#165]
Would be nice to see some simple ligatures examples in your code examples.

!= :=  and ->

**Comment by chrissimpkins** [comment]
The ones included in the bottom line of the top specimen were decided by @kevinSuttle in https://github.com/chrissimpkins/codeface/pull/163 

Suggest that we modify examples shown there?  Would like to limit this to a single line in the specimen if possible.

**Comment by chrissimpkins** [comment]
Also see new PR #164 that just came in today

**Comment by chrissimpkins** [comment]
Added to last line of the top black/white text specimen.  Currently in `development` branch and you can view the new gallery images here:

https://github.com/chrissimpkins/codeface/tree/development

Here is an example of a font that renders with ligatures:

https://github.com/chrissimpkins/codeface/tree/development#fira-code--

Waiting on feedback from #164 and will merge to master so that this is live.

**Comment by chrissimpkins** [comment]
now in images in master!
